config/config.exs

```diff
@@ -4,4 +4,6 @@ use Mix.Config
 config :logger, :console, colors: [enabled: false]
 
 # Use higher stacktrace depth.
-config :phoenix, :stacktrace_depth, 20
\ No newline at end of file
+config :phoenix,
+  stacktrace_depth: 20,
+  filter_parameters: ["password"]
```

Comment:
That configuration won't be available until after the app is compiled, so it makes sense.